src/ui/Button/Button.tsx

```diff
@@ -4,7 +4,14 @@ import { styled } from '../../stitches.config';
 import { modifyVariantsForStory } from '../type-utils';
 
 export const Button = styled('button', {
+  '& img': {
+    paddingRight: '$sm',
+  },
+  '& svg': {
+    margin: '$sm',
+  },
   px: '$md',
+  py: '$sm',
```

Comment:
```suggestion
```

I intentionally set zero vertical padding on buttons so that their contents don't change the button heights